CHANGELOG.md

```diff
@@ -11,9 +11,11 @@ and this project adheres to [Semantic Versioning][semver].
 
 - Clone target repositories to temp ([412, 418])
 - Add architecture overview documentation ([405])
+- Reimplemented get_file_details function to not rely on old securesystemslib functions ([420])
```

Comment:
We typically sort changelog entries from newer to older. Everything else looks good!